packages/components/transcript-editor/index.js

```diff
@@ -451,6 +462,7 @@ class TranscriptEditor extends React.Component {
         handleAnalyticsEvents={this.props.handleAnalyticsEvents}
         handleAutoSaveChanges={this.handleAutoSaveChanges}
         autoSaveContentType={contentFormat}
+        isAutoSave={typeof this.props.isAutoSave ==='boolean'? this.props.isAutoSave : true}
```

Comment:
Same for this kinda thing too :)

Reply:
as in, set it up in your parent - `this.state.isAutosave: true` or something in the parent